client/plugins/vuetify-snackbar/index.js

```diff
@@ -10,6 +10,7 @@ export const install = (Vue, { parent = document.body } = {}) => {
   const $snackbar = {
     show: (msg, options) => queue.add(() => vm.show(msg, options)),
     close: () => vm.close,
+    primary: (msg, options) => $snackbar.show(msg, { ...options, color: 'primary' }),
```

Comment:
success, info  and warning have semantics while primary doesn't.  Maybe we should config info to primary color or figure out better name for 'primary'. @vladimyr ?